lib/create-contentful-api.js

```diff
@@ -89,6 +90,7 @@ export default function createContentfulApi ({http, getGlobalOptions}) {
    * .catch(console.error)
    */
   function getSpace () {
+    http.defaults.baseURL = getGlobalOptions().environmentBaseUrl
```

Comment:
I am pretty sure this is the wrong url, since `environmentBaseUrl` does contain `/environments/master`

Correct? So it should be `getGlobalOptions().spaceBaselUrl `

Reply:
yeah, I was under the assumption that you can access the env, but you don't